packages/wallet/ui/src/components/AppBar.jsx

```diff
@@ -63,13 +66,21 @@ const useStyles = makeStyles(theme => ({
 }));
 
 // Exported for testing only.
-export const AppBarWithoutContext = ({ connectionComponent }) => {
+export const AppBarWithoutContext = () => {
   const theme = useTheme();
   const classes = useStyles(theme);
 
-  const [dialogOpen, setDialogOpen] = useState(false);
-  const handleClosed = () => {
-    setDialogOpen(false);
+  const [areTermsAccepted, setAreTermsAccepted] = useState(
```

Comment:
for future, consider sticking to `is` convention. e.g. `isDisclaimerAccepted`.

Reply:
In the style guide I learned from, both 'is' and 'are' are okay depending on if it's plural, also 'can', 'should', 'has', etc. I forget the grammatical term for this type of phrase though.